src/components/NavBar /NavBar.jsx

```diff
@@ -0,0 +1,102 @@
+import {
+  Box,
+  Button,
+  Drawer,
+  DrawerContent,
+  DrawerHeader,
+  DrawerOverlay,
+  IconButton,
+  useDisclosure,
+} from "@chakra-ui/react";
+import React from "react";
+import { BiBookBookmark as ShoppingCartIcon } from "react-icons/bi";
+import SectionSeperator from "./SectionSeperator";
+import ShoppingCartItem from "./ShoppingCartItem";
+
+const NavBar = () => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const [shoppingCartItems] = React.useState([
+    { planName: "Ladder T-walls", imageNum: 2, commentNum: 8 },
+    { planName: "Ladder T-walls", imageNum: 3, commentNum: 6 },
+    { planName: "Ladder T-walls", imageNum: 1, commentNum: 2 },
+    { planName: "Ladder T-walls", imageNum: 4, commentNum: 5 },
+    { planName: "Ladder T-walls", imageNum: 6, commentNum: 7 },
+  ]);
+
+  const handleClose = () => {
+    onClose();
+  };
+
+  return (
+    <div>
+      <nav>
+        <div style={NavBarContainer}>
+          <div style={NavBarElement}>Digital Library</div>
+          <div style={NavBarElement}>Project Plan Builder</div>
+          <div style={NavBarElementCart}>
+            <IconButton onClick={onOpen} icon={<ShoppingCartIcon />} />
+            <div style={NavBarElement}>Logout</div>
+          </div>
+        </div>
+      </nav>
+
+      <Drawer isOpen={isOpen} onClose={handleClose}>
+        <DrawerOverlay />
+        <DrawerContent justifyContent="space-between">
+          <Box>
+            <DrawerHeader display="flex" justifyContent="center">
+              Atlanta Construction 47
+            </DrawerHeader>
+            <SectionSeperator />
+            <ui>
+              {shoppingCartItems.slice(0, 3).map((item, index) => (
+                <ShoppingCartItem
+                  key={index}
+                  planName={item.planName}
+                  imageNum={item.imageNum}
+                  commentNum={item.commentNum}
+                  shoppingCartItems={item.shoppingCartItems}
+                  // setShoppingCartItems={item.setShoppingCartItems}
+                />
+              ))}
+            </ui>
+          </Box>
+          <Box display="flex" justifyContent="center">
+            <Button
+              onClick={handleClose}
+              background="#6D6E70"
+              borderRadius="32px"
```

Comment:
please try not to use pixel values because they are absolute units and may mess up with different screen sizes of different monitors. Look into chakra Ui's relative sizing `xs, sm, md, lg, xl, 2xl` etc. If a component does not support one of those values then use `em` or `rem` units. Only the last resort should be using pixel values or unless you cant achieve what you want to achieve with the relative units.

Reply:
This applied to all the components where you used pixel values.